src/Resources/views/Form/roles_matrix_list.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% for role, attributes in roles|sort %}
-    <li>{{ form_widget(attributes.form, {label: attributes.role_translated, value: attributes.role}) }}</li>
+    <li>{{ attributes.form is defined ? form_widget(attributes.form, {label: attributes.role_translated, value: attributes.role}) }}</li>
```

Comment:
Shouldnt the whole li be hidden if the form is not defined ?

(Also are you able to add a functionnal test for this bug ?)

Reply:
I updated the code to surround the `<li>` element.  I think it will take me some time to properly setup for creating tests.  I don't know if you should wait on me for that though.

Reply:
I wont be able to patch this before sunday/monday anyway since I currently doesnt have a computer.

Reply:
No worries.  I was able to add an initial functional test.